.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+**Checklists** 
+- [ ] Added impacted package name to the issue description
+
+**Packages impacted by this PR:**
+
+
+**Issues associated with this PR:**
```

Comment:
I wonder if there's some way to provide a starting point that encourages linking issues using [GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

Maybe like `Resolves #<issue>` ? And I also wonder if it should be on the bottom as it's less important to readers of the PR